### PR TITLE
feat: document new context pad alpha feature

### LINKED ISSUE #3243: Document New Context Menu
Once we've rolled out the new context menu so it can be enabled as an alpha feature we have to have it documented somewhere so users can find out what they're enabling before doing so. The documentation will be accessible from where the feature is enabled or disabled.

image

An existing example of a documented alpha feature is Play: https://docs.camunda.io/docs/components/modeler/web-modeler/play-your-process/

---

Related to https://github.com/camunda/product-hub/issues/1505

### PR DESCRIPTION
**Pull request by philippfromme**
Closes #3243

## When should this change go live?

- [x] This change is not yet live and should not be merged until https://github.com/camunda/web-modeler/pull/7847 was merged and released (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/new-context-pad.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+id: new-context-pad
+title: New context pad
+description: The new context pad is a feature that allows you to easily add elements to your diagram and to perform actions on elements.
+---
+
+<span class="badge badge--cloud">Camunda 8 only</span>
+<span class="badge badge--beta">Alpha</span>
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/new-context-pad.md:8]: Will adjust this in a separate PR 👍

**Review by christinaausley — COMMENTED** [review]
Technical review in latest commit, but I have a few thoughts.

I would note _how_ the user can see the context pad -- does it automatically show up? Can you move it around? Can you minimize it or re-open it?

This rolls into my next comment. For accessibility purposes, we can't rely on the images to demonstrate the capabilities of the context pad. We need to **tell** the user what buttons/icons to click on for each feature.

**Comment by philippfromme** [comment]
@christinaausley I see you removed the _new_ part. We have to account for the fact that there is an existing context pad. Now the page looks like it's documenting the existing context pad, which will confuse users who see the old context pad. We don't have this issue with other alpha features like Play, where the entire feature is new. How should we handle that?

**Comment by christinaausley** [comment]
@philippfromme As https://github.com/camunda/web-modeler/pull/7847 was merged, would you like to see this go out with release tomorrow?

File: docs/components/modeler/web-modeler/new-context-pad.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/modeler/web-modeler/new-context-pad.md:8]: Can add an alpha badge here now, as added in https://github.com/camunda/camunda-docs/pull/3357/files.

**Comment by philippfromme** [comment]
@christinaausley I agree that we don't want to refer to things as _new_ whenever possible. In the future I'd like to see alpha features to be documented in an entirely separate section that establishes the necessary context so we don't have to use _new_. And yes, it would be great for this to be released.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**